VideoLocker/src/main/java/org/edx/mobile/base/BaseFragmentActivity.java

```diff
@@ -115,8 +118,6 @@ protected void onStart() {
         super.onStart();
         isActivityStarted = true;
 
-        handler.sendEmptyMessage(MSG_TYPE_TICK);
-
         PrefManager pmFeatures = new PrefManager(this, PrefManager.Pref.FEATURES);
 
         boolean enableSocialFeatures = NetworkUtil.isSocialFeatureFlagEnabled(this, environment.getConfig());
```

Comment:
There are quite a few formatting issues in these files, so as this is not related to my changes, I am leaving it for now. Maybe all the files should be auto-formatted at some point later.